cysync/src/store/provider/syncProvider/executors/balance.ts

```diff
@@ -84,10 +84,12 @@ export const processResponses = async (
 
     const balance = new BigNumber(balanceRes.data);
 
-    await xpubDb.updateTotalBalance(item.xpub, item.coinType, {
-      balance: balance.toString(),
-      unconfirmedBalance: '0'
-    });
+    await coinDb.updateTotalBalance(
```

Comment:
not more than 1-2 params, prefer named params, that'll make it readable

Reply:
Didnt get you, could you elaborate?

Reply:
Thanks for the suggestion, its done ✅